Mathlib/LinearAlgebra/BilinearForm/TensorProduct.lean

```diff
@@ -59,36 +59,36 @@ def tensorDistrib : BilinForm A M₁ ⊗[R] BilinForm R M₂ →ₗ[A] BilinForm
 theorem tensorDistrib_tmul (B₁ : BilinForm A M₁) (B₂ : BilinForm R M₂) (m₁ : M₁) (m₂ : M₂)
     (m₁' : M₁) (m₂' : M₂) :
     tensorDistrib R A (B₁ ⊗ₜ B₂) (m₁ ⊗ₜ m₂) (m₁' ⊗ₜ m₂')
-      = B₂ m₂ m₂' • B₁ m₁ m₁' :=
-  rfl
+      = B₂ m₂ m₂' • B₁ m₁ m₁' := LinearMap.tensorDistrib_tmul (toLin B₁) (toLin B₂) m₁ m₂ m₁' m₂'
```

Comment:
I don't think this is good style: if the conclusion is broken across multiple lines, then the proof should get its own line; that is
```
example :
    something = else := rfl
```
is fine, as is
```
example :
    something = else :=
  rfl
```
but
```
example :
    something
      = else := rfl
```
is not, as it is much harder to see where the statement ends

Reply:
Thanks. I've put this back to rfl.